CONTRIBUTING.md

```diff
@@ -123,35 +132,65 @@ cd ocaml && make -j9 world.opt && make install && cd ..
 
 ## Contributing to the runtime
 
-BuckleScript runtime implementation is currently a mix of OCaml and JavaScript. (`jscomp/runtime` directory). The JavaScript code is defined in the `.ml` file using the `bs.raw` syntax extension.
+BuckleScript runtime implementation is currently a mix of OCaml and JavaScript.
+(`jscomp/runtime` directory).
```

Comment:
actually it is not a mix any more, it is pure ocaml with some raw js embedded

Reply:
edited that as well